bokehjs/test/models/annotations/color_bar.coffee

```diff
@@ -25,6 +25,7 @@ ColorBar = proxyquire('../../../src/coffee/models/annotations/color_bar',
                       {"../../core/util/text": textStub})
 
 describe "ColorBar module", ->
+  return
```

Comment:
There's actually a simple fix for this:

```
diff --git a/bokehjs/test/models/annotations/color_bar.coffee b/bokehjs/test/models/annotations/color_bar.coffee
index 60aa69a..73cacad 100644
--- a/bokehjs/test/models/annotations/color_bar.coffee
+++ b/bokehjs/test/models/annotations/color_bar.coffee
@@ -47,9 +47,8 @@ describe "ColorBar module", ->

     beforeEach ->
       # Stub solver computed values with deterministic frame height and width
-      frame_stub = sinon.stub(@plot.plot_canvas.frame, 'get')
-      frame_stub.withArgs('height').returns(500)
-      frame_stub.withArgs('width').returns(500)
+      sinon.stub(@plot.plot_canvas.frame, 'height').returns(500)
+      sinon.stub(@plot.plot_canvas.frame, 'width').returns(500)

     describe "ColorBar.Model._title_extent method", ->
```

Do you want to include or should I follow up in another PR?